config/config.exs

```diff
@@ -1,10 +1,9 @@
 use Mix.Config
 
-config :logger, level: :info
+config :logger, level: :debug
```

Comment:
Wouldn't `:info` be better?  Not a deal breaker, just curious.

Reply:
Probably. I think this was something that I did to debug something in test.